elastalert/alerts.py

```diff
@@ -256,7 +256,7 @@ def get_aggregation_summary_text__maximum_width(self):
         return 80
 
     def get_aggregation_summary_text(self, matches):
-        text = ''
+        text = self.rule.get('summary_prefix', '')
```

Comment:
Just realized that this needs to go into the if statement. Otherwise we might get a lone summary_prefix.
```
text = ''
if 'aggregation' in self.rule and .....
    text = self.rule.get('summary_prefix', '')
```

Reply:
Good observation！